server/src/main/java/org/opensearch/common/util/CuckooFilter.java

```diff
@@ -83,7 +81,7 @@ public class CuckooFilter implements Writeable {
     private static final int MAX_EVICTIONS = 500;
     static final int EMPTY = 0;
 
-    private final XPackedInts.Mutable data;
+    private final Packed64 data;
```

Comment:
I think this class is package protected in Lucene (but XPackedInts is not):

```
class Packed64 extends PackedInts.MutableImpl {
```

Reply:
Initially coming from https://github.com/opensearch-project/OpenSearch/tree/main/server/src/main/java/org/apache/lucene/util/packed, the XPackedInts has the dependency of XPacked64 and XPacked64SingleBlock which is only used in this CuckooFilter.

Reply:
Oh sorry, I haven't seen you copied it as inner class, got it